shared/FamilyHubs.Notification.Api.Contracts/MessageDto.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FamilyHubs.Notification.Api.Contracts;
+
+public record MessageDto : DtoBase<long>
+{
+    public required string RecipientEmail { get; set; }
+    public required string TemplateId { get; set; }
+    public Dictionary<string, string> TemplateTokens { get; set; } = new Dictionary<string, string>();
+    public DateTime? Created { get; set; }
+
+    public override int GetHashCode()
```

Comment:
GetHashCode and Equals are provided for free by using a record.

Reply:
Yes they are, but tests such as WhenComparingTwoObjectsWithSameValuesThenReturnsTrue and WhenComparingTwoObjectsIgnorePrimaryAndForeignKeyThenReturnsTrue fail if I don't overload those methods.